Automattic/wp-calypso
# Focused-Launch: Fix scroll to top when transitioning between routes

We want each view in the Focused Launch to scroll to the top when transitioning between routes since information is "hidden" from the user's view when the scrolling position is preserved. This PR introduces a ScrollToTop component that is triggered on every route transition in the Focused Launch. It does the heavy-scroll-to-top-lifting on every route transition by hooking into the react-router-doms features.

#### Changes proposed in this Pull Request

* Every view in the Focused Launch should be scrolled to the top of the page automatically

#### Testing instructions
* Visit `/page/SANDBOXED_UNLAUNCHED_SITE_ID_CREATED_WITH_START.wordpress.com/home?flags=create/focused-launch-flow-calypso`. Focused Launch modal should appear
* Re-size the window reducing the height so a vertical scrollbar appears.
* Scroll down as far as you can, but make sure "View all domains" is still visible.
* Click on the _"View all domains"_ link. The new view is scrolled to the top of the page, showing all information.

Fixes #47325
